### DISCUSSION TITLE
Is there an API Endpoint for selecting if a digital listing is AI generated or made by the user?

### DISCUSSION THREAD

**Discussion by newParrot** [etsy/open-api#1340] [Q&A]
I am looking to help streamline some listings, but can't seem to find any updated endpoints for the Etsy API regarding selecting whether a listing is AI generated or made by the user. Not being able to do so causes unintended interactions on the listing like wrong files, not being able to see/edit the listing download files, etc. Does anyone have any experience with this?

![image](https://github.com/user-attachments/assets/a9f1bfc9-34b8-473b-83e2-9eeaf17ac23b)

**Comment by SrdjanPaunovic** [comment]
+1 here
